oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
Can you add a note in the docstring about these credentials needing to be refreshed before usage?